.travis.yml

```diff
@@ -2,14 +2,16 @@ language: node_js
 sudo: required
 node_js:
   - "0.10"
+  - "4"
   - "4.4.3"
+  - "6"
+  - "8"
```

Comment:
lts/* will be better here. It my suggest that we support 8 :)

Reply:
@wtrocki not sure I follow, do you mean we can put "*" instead of "8"?